docs/proposals/scenarios/Smart_Coding/Smart Coding benchmark suite Proposal_zh.md

```diff
@@ -0,0 +1,172 @@
+# 背景
```

Comment:
The proposals are not needed in this implementation PR. Shall keep the proposal in the proposal PR, i.e., https://github.com/kubeedge/ianvs/pull/120

Besides, all the content written in Chinese should be translated into English to facilitate international understanding and usage.